Documentation/2022 12 21 minutes

JesusSierralaya opened the pull request:
please review or fill missing parts of today´s meeting

robertomediavilla committed:
first template draft (2nd attempt)

DaniMori committed:
Reorders paragraphs into the "blank field" prompt

DaniMori committed:
Delete prompt

DaniMori committed:
Updates "Introduction" paragraph.

DaniMori committed:
Updates "Context" paragraph (WIP)

JesusSierralaya committed:
minutes 2022-12-21 Jesus version

robertomediavilla commented on doc/minutes/2022-12-21.md:
```diff
@@ -0,0 +1,70 @@
+
+-   **Everybody**: Must approve the previous changes
+
+## About what we need to put about ourselves
```
I suggest rename this section "On the team description section"

robertomediavilla reviewed (dismissed):
Thanks Jesús. I just added a minor suggestion but won't request changes

JesusSierralaya committed:
Added Robert's suggestion and fixed other superficial mistakes

JesusSierralaya committed:
Added Robert's suggestion and fixed other superficial mistakes

JesusSierralaya committed:
solves mistakes 2 versions

Merge branch 'documentation/2022-12-21-minutes' of https://github.com/CCOMS-UAM/yos-award into documentation/2022-12-21-minutes

# Conflicts:
#	doc/minutes/2022-12-21.md

JesusSierralaya committed:
removes the commit error

DaniMori committed:
Completes first draft of the 21-12-2022 minutes

DaniMori commented:
@JesusSierralaya I can't request a review from your side, but you can "review and comment" anyway

robertomediavilla commented on doc/minutes/2022-12-21.md:
```diff
@@ -0,0 +1,95 @@
+
+#### Debate
+
+-   Suggested that the "team description"" would better include a description of
```
"team description", one " extra

DaniMori replied:
Thanks! Fixed in baf61bb8cfc2219a1ff0dfff6198db5a1844edd4

robertomediavilla reviewed (dismissed):
Thaaaanks!!

DaniMori reviewed (commented):
@JesusSierralaya it seems like I branched from `drafting/description` instead of `main`. Hopefully it doesn't create any conflict...

JesusSierralaya reviewed (commented):
I approve the previous changes!

DaniMori committed:
Corrects erratum in the use of quotes

DaniMori commented:
@robertomediavilla and @Leafrancia please review and approve latest version, thanks!
